pom.xml

```diff
@@ -928,6 +929,30 @@
         <version>${netty.version}</version>
         <classifier>osx-x86_64</classifier>
       </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
What's the license of this code? if it's the same as netty, I think we're all good. Otherwise need to check it and possibly update LICENSE and licenses/

Reply:
@srowen `netty-tcnative-boringssl-static` seems to be apache2 licensed: https://github.com/netty/netty-tcnative/blob/main/LICENSE.txt

which is the same as `netty` as far as I Can tell: https://github.com/netty/netty/blob/4.1/LICENSE.txt